flyway/scxa/migrations/V15__scxa_add_coords_param_index_not_null.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE scxa_coords ALTER COLUMN json_parameterisation SET NOT NULL;
```

Comment:
I’m not a fan of Hungarian notation. Maybe we could just use the same `parameterisation` field name if we’re not going to keep the old column?